src/main/resources/application-local.properties

```diff
@@ -0,0 +1,98 @@
+spring.jpa.database=POSTGRESQL
+spring.datasource.platform=postgres
+spring.jpa.properties.hibernate.hbm2ddl.import_files_sql_extractor=org.hibernate.tool.hbm2ddl.MultipleLinesSqlCommandExtractor
+spring.jpa.hibernate.hbm2ddl.import_files_sql_extractor=org.hibernate.tool.hbm2ddl.MultipleLinesSqlCommandExtractor
+spring.jpa.hibernate.ddl-auto=none
+
+spring.http.gzip.enabled=true
+
+#spring.jpa.properties.hibernate.format_sql=true
+
+spring.jpa.properties.hibernate.order_updates=true
+spring.jpa.properties.hibernate.batch_versioned_data=true
+
+spring.flyway.enabled=true
+spring.flyway.table=schema_version
+
+server.compression.mime-types=application/json,application/xml,text/html,text/xml,text/plain
+
+spring.jpa.hibernate.id.new_generator_mappings=true
+spring.jpa.hibernate.use-new-id-generator-mappings=true
```

Comment:
Deprecated Hibernate no longer supports disabling the use of new ID generator mappings.

Whether to use Hibernate's newer IdentifierGenerator for AUTO, TABLE and SEQUENCE. This is actually a shortcut for the "hibernate. id. new_generator_mappings" property. When not specified will default to "true".